lib/boinc_win.h

```diff
@@ -148,6 +148,8 @@
 #define SECURITY_WIN32
 #endif
 
+#include <cstddef>
```

Comment:
is this really required?

Reply:
it's for size_t https://github.com/BOINC/boinc/blob/master/lib/boinc_win.h#L164C9-L164C15